.travis.yml

```diff
@@ -7,9 +7,9 @@ rvm:
   - jruby-head
   - 2.1.10
   - 2.2.9
```

Comment:
Probably we should update Ruby 2.2 to 2.2.10 also.
https://www.ruby-lang.org/en/news/2018/03/28/ruby-2-2-10-released/

Reply:
Oops, thank you. I'll fix it.